### DISCUSSION TITLE
Getting Tasks by name always results in TaskConnectionStatus of NotActive

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#17464] [General]
We are implementing some monitoring code, so I am querying subscriptions to see if the subscription is active or not.

When I use 
```
var op = new GetOngoingTaskInfoOperation("Subscriptions_NewSalesData", OngoingTaskType.Subscription);
var ongoingTask = store.Maintenance.Send(op);
```

`ongoingTask.TaskConnectionStatus` is **always** `NotActive`.

However, if I use the TaskId
```
var op = new GetOngoingTaskInfoOperation(22 OngoingTaskType.Subscription);
var ongoingTask = store.Maintenance.Send(op);
```

`ongoingTask.TaskConnectionStatus` shows the actual status.

As a stop gap, I'm querying the task by name, getting the id, and requerying by that id.

Am I doing something wrong here?

**Comment by garayx** [comment]
Hi

Thanks for reporting this,

https://issues.hibernatingrhinos.com/issue/RavenDB-21541

**Reply by garayx** [reply]
@ItsRobbAllen  the fix is merged (https://github.com/ravendb/ravendb/pull/18075) and available in the latest nightly
